### PR TITLE
docs(operators): add documentation for distinctUntilChanged

### LINKED ISSUE #90: Add docs for operator : filtering - distinctUntilChanged
Add docs for operator distinctUntilChanged under group filtering 
 
Below is minimum details needed 
 
- [ ] name 
- [ ] operatorType 
- [ ] signature 
- [ ] parameters 
- [ ] short description 
- [ ] Technical description 
- [ ] Visual example  (Marble diagram) 
- [ ] Real world example (Non-framework) 
- [ ] Related Operators

### PR DESCRIPTION
**Pull request by btroncone**
Closes #90

### CODE REVIEW

File: src/operator-docs/filtering/distinctUntilChanged.ts [hunk 1]
```diff
@@ -1,6 +1,95 @@
...
+      name: 'An example using a compare function',
+      code: `
+      Rx.Observable.of(
+         { age: 4, name: 'Foo'},
+         { age: 7, name: 'Bar'},
+         { age: 5, name: 'Foo'},
+         { age: 6, name: 'Foo'}
+        )
+        .distinctUntilChanged((p, q) => p.name === q.name)
+         // displays:
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/filtering/distinctUntilChanged.ts]: Can we move the displays block after the subscribe to make the chain a bit more clear ?

File: src/operator-docs/filtering/distinctUntilChanged.ts [hunk 2]
```diff
@@ -1,6 +1,95 @@
...
+      </ul>
+    `
+  },
+  examples: [
+    {
+      name: 'A simple example with numbers',
+      code: `
+      Rx.Observable.of(1, 1, 2, 2, 2, 1, 1, 2, 3, 3, 4)
+       .distinctUntilChanged()
+       // displays
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/filtering/distinctUntilChanged.ts]: Can we move the displays block after the subscribe to make the chain a bit more clear ?

File: src/operator-docs/filtering/distinctUntilChanged.ts (hunk 1 above)
Comment (btroncone) [reply src/operator-docs/filtering/distinctUntilChanged.ts]: Yep, I updated the JSBin links and removed from `ts` files as this is just used for copy / paste.

**Comment by ladyleet** [comment]
@btroncone woot thanks brian! :) and thx @ashwin-sureshkumar and @sumitarora for reviewing <3

**Pull request merged**